src/nbl/video/IOpenGL_LogicalDevice.h

```diff
@@ -261,6 +263,16 @@ namespace impl
 
             core::smart_refctd_ptr<IDriverMemoryAllocation> buf;
         };
+        struct SRequestSetDebugName
+        {
+            static inline constexpr E_REQUEST_TYPE type = ERT_SET_DEBUG_NAME;
+            using retval_t = void;
+
+            GLenum id;
+            GLuint object;
+            GLsizei len;
+            std::string label;
```

Comment:
OpenGL only supports max 256 character long debug names

Reply:
256 is min max afaik

Reply:
still play it safe, also having `std::string` floating around in GPU API objects is awful (malloc every !*("$@ where)

Reply:
yeah, thats right